### DISCUSSION TITLE
AI Unbound

### DISCUSSION THREAD

**Discussion by jtmuller5** [jtmuller5/The-HustleGPT-Challenge#32] [Ventures]
![image](https://user-images.githubusercontent.com/47997351/226370270-96a00281-8844-4b98-9815-59cdf7849dce.png)

Website: [AI Unbound](https://www.dreamulatorgame.com/)
Tagline: A game crafted by both human and artificial minds
Twitter: [@dreamulatorgame](https://twitter.com/dreamulatorgame)

**Comment by rafalkorbaczynski** [comment]
Hey! In addition to following along the adventure on Twitter, you can check out the detailed GPT output which I'm creating the game around [in this blog post.](https://www.dreamulatorgame.com/aigame) 
If you prefer videos, you can check out my day 0 devlog [here, on my YouTube channel!](https://youtu.be/UkmkIUZPDjE) 
Look out for more updates here, on Twitter and YouTube. This evening, I start work on sprites and graphics :)
